src/ui-config/networksConfig.ts

```diff
@@ -340,4 +340,21 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
       url: 'https://app.multichain.org/#/router',
     },
   },
+  [ChainId.xdai]: {
+    name: 'Gnosis Chain',
+    publicJsonRPCUrl: ['https://rpc.gnosis.gateway.fm', 'https://rpc.gnosischain.com'], // todo validate https://docs.gnosischain.com/tools/rpc/
```

Comment:
can we get ANKR url here? and get rid of this todo?

`https://rpc.ankr.com/gnosis` maybe replace gateway.fm with this?